SoftServeCarMobile/coverage/lcov-report/SoftServeCarMobile/api-service/Interceptor.tsx.html

```diff
@@ -0,0 +1,206 @@
+
+<!doctype html>
+<html lang="en">
+
+<head>
+    <title>Code coverage report for SoftServeCarMobile/api-service/Interceptor.tsx</title>
+    <meta charset="utf-8" />
+    <link rel="stylesheet" href="../../prettify.css" />
+    <link rel="stylesheet" href="../../base.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="../../favicon.png" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <style type='text/css'>
+        .coverage-summary .sorter {
+            background-image: url(../../sort-arrow-sprite.png);
+        }
+    </style>
+</head>
+    
+<body>
+<div class='wrapper'>
+    <div class='pad1'>
+        <h1><a href="../../index.html">All files</a> / <a href="index.html">SoftServeCarMobile/api-service</a> Interceptor.tsx</h1>
+        <div class='clearfix'>
+            
+            <div class='fl pad1y space-right2'>
```

Comment:
I've seen these duplications in different files. Can we omit it? Would be nice to have some base coverage component or etc.